main.tf

```diff
@@ -0,0 +1,68 @@
+resource "null_resource" "default" {
+  triggers = {
+    id = "${lower(format("%v-%v-%v-%v", var.namespace, var.stage, var.name, "assets"))}"
```

Comment:
Why define a new `null_resource` when you can just use `tf_label` and append the specifier?